sagemaker-python-sdk/tensorflow_script_mode_quickstart/tensorflow_script_mode_quickstart.ipynb

```diff
@@ -91,19 +99,7 @@
     "args = parser.parse_args()\n",
     "\n",
     "```\n",
-    "When SageMaker training finishes, it deletes all data generated inside the container with exception of the directories `_/opt/ml/model_` and `_/opt/ml/output_`. To ensure that model data is not lost during training, training scripts are invoked in SageMaker with an additional argument `--model_dir`, that needs to be handle by the training script.\n",
-    "\n",
-    "We need to replace the argument `--save_dir` with the required argument `--model_dir`:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# this command will replace data_dir by model_dir in the training script\n",
-    "!sed -i 's/save_dir/model_dir/g' char-rnn-tensorflow/train.py"
+    "When SageMaker training finishes, it deletes all data generated inside the container with exception of the directories `_/opt/ml/model_` and `_/opt/ml/output_`. To ensure that model data is not lost during training, training scripts are invoked in SageMaker with an additional argument `--model_dir`, that needs to be handle by the training script.\n"
```

Comment:
"...in SageMaker with an additional argument `--model_dir`. The training script should save the model data that results from the training job to this directory."